app/models/log.js

```diff
@@ -21,7 +23,7 @@ var Request = Ember.Object.extend({
   },
 
   handle(body, status, xhr) {
-    if (config.pro) {
+    if (config.featureFlags.pro) {
```

Comment:
Yeah, I've tried to keep the changes as small as possible to avoid getting overwhelmed. Definitely think this should happen in the future though.